### PR TITLE
[Communication] [Chat] 2024 03 01 preview

### PR DESCRIPTION
**Pull request by emlynmac**
Copy from 2023 11 15 preview. 
Behavioural change required a new version.

No API changes

The default version should remain as the latest preview, not this version.

### CODE REVIEW

File: specification/communication/data-plane/Chat/preview/2024-03-01-preview/communicationserviceschat.json [hunk 1]
```diff
@@ -0,0 +1,1817 @@
...
+          "Post typing event to a thread": {
+            "$ref": "./examples/Messages_SendTypingNotification.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationUserIdentifierModel": {
+      "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationUserIdentifierModel"
```
Comment (jpeng-ms) [inline specification/communication/data-plane/Chat/preview/2024-03-01-preview/communicationserviceschat.json:1202]: we left common sdk update in this hotfix after checking with @waynemo . The upcoming API version will have common SDK updates are:
- 2024-03-07
- 2024-03-15-preview

**Review by JeffreyRichter — APPROVED** [review]

File: specification/communication/data-plane/Chat/preview/2024-03-01-preview/communicationserviceschat.json [hunk 2]
```diff
@@ -0,0 +1,1817 @@
...
+    },
+    "CommunicationCloudEnvironment": {
+      "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationCloudEnvironmentModel"
+    },
+    "MicrosoftTeamsUserIdentifierModel": {
```
[the remark this answers is not in the document]
Comment (jimchou-dev) [reply specification/communication/data-plane/Chat/preview/2024-03-01-preview/communicationserviceschat.json:1207]: the hotfix is for the Teams visitor, not for bumping common to include TeamsApp user, ignore my comment

**Pull request merged**